lua/lint/linters/djlint.lua

```diff
@@ -0,0 +1,19 @@
+local pattern = [[([^:]+):(%d+):(%d+):(%d+): (.*)]]
+local groups = { 'file', 'lnum', 'col', 'code', 'message' }
+
+local defaults = {
+  ['source'] = 'djlint',
+  ['severity'] = vim.diagnostic.severity.INFO
+}
+
+return {
+  cmd = 'djlint',
+  stdin = false,
+  args = {
+    '--linter-output-format',
+    '{filename}:{line}:{code}: {message}',
```

Comment:
The pattern doesn't match this format specification. Are you sure this is working?

Reply:
I'm pretty sure it's correct, since the `{line}` formatting symbol returns a `line:col` formatted string.

I'm not sure about how the arguments are sent, because the `{filename}:{line}:{code}: {message}` should be a string by itself.

In essence the command would be `djlint --linter-output-format '{filename}:{line}:{code}: {message}'  filename`.